### DISCUSSION TITLE
KOT Printout Improvements

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#456] [Ideas]
Hi Team,

Please provide an option to the admin for making changes in the KOT printout. 

1) logo can be removed or added on KOT printout
2) order type should be mentioned on KOT printout: dine-in, delivery, takeout or if any
3) if it is not dine-in, then table row is not required to be printed on  KOT printout instead it is showing empty now
4) lots of empty spaces in the top

![image](https://user-images.githubusercontent.com/16856248/147848388-f32b5a71-feac-441a-a40a-929528ae668e.png)

Current KOT above from NexoPOS

**Comment by Blair2004** [comment]
Hi, 
Alright, we'll consider those adjustments.
Thanks
